src/null-safety/index.md

```diff
@@ -3,13 +3,36 @@ title: Sound null safety
 description: Information about Dart's null safety feature
 ---
 
-The Dart language now supports sound null safety!
+The Dart language comes with sound null safety.
+
+Null safety prevents errors that result from unintentionel access
+of variables set to `null`.
+For example, if a method expects an integer but receives `null`,
+your app causes a runtime error. This type of error, a null dereference error,
+can be difficult to debug.
+
+With sound null safety variables are 'non-nullable' by default.
+You can specify that the type of a variable is nullable,
+and only then can they contain a `null`.
```

Comment:
**question:** Is this redundant? Does saying a variable type is nullable the same as saying the variable can contain `null`?

Reply:
It's not redundant, but I can see the confusion.

All variables have a type (e.g. `int` or `String`). Prior to null safety, those types were all nullable, so a variable could contain either a value of the specified type (e.g. 42 or "Foo") OR they could contain a null.

With null safety, all types are either non-nullable (e.g. `int` or `String) or nullable (e.g. `int?` or `String?`). Variables of the former can never contain null, while the latter still can.

Should I try to work this into the introduction (which would also address the comment above)?

Reply:
I think addressing the previous comment would essentially do this.  That is, if your previous sentence looked like "'non-nullable' by default, which means that they are not allowed to contain `null`", then this sentence in turn de facto defines what nullable means.

Reply:
OK, I took a stab at that, PTAL